pinot-server/src/main/java/org/apache/pinot/server/api/resources/TablesResource.java

```diff
@@ -125,8 +125,8 @@ public class TablesResource {
   //swagger annotations
   @ApiOperation(value = "List tables", notes = "List all the tables on this server")
   @ApiResponses(value = {
-      @ApiResponse(code = 200, message = "Success", response = TablesList.class),
-      @ApiResponse(code = 500, message = "Server initialization error", response = ErrorInfo.class)
+      @ApiResponse(code = 200, message = "Success", response = TablesList.class), @ApiResponse(code = 500, message =
```

Comment:
Although it is done by auto reformat, let's not reformat the `@ApiResponse` because it makes it less readable. Same for other APIs